aziotd/src/main.rs

```diff
@@ -156,9 +156,11 @@ where
             hyper::Request<hyper::Body>,
             Response = hyper::Response<hyper::Body>,
             Error = std::convert::Infallible,
-        > + Clone
+        >
+        + Clone
         + Send
-        + 'static,
```

Comment:
`Clone + Send + Sync + 'static` in that order, to be consistent.

Also, can you check if you really need the `Sync` bound? I'm surprised you do, given that `server` is essentially being used in the same way as before.

Reply:
Didn't need it. I need to clean this up.